packages/switch/stories/index.stories.js

```diff
@@ -0,0 +1,90 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { LitElement } from '@lion/core';
+
+import { LocalizeMixin } from '@lion/localize';
+
+import '../lion-input-switch.js';
+import '@lion/form/lion-form.js';
+
+storiesOf('Switches|Switch', module)
+  .add(
+    'Default',
+    () => html`
+      <lion-input-switch></lion-input-switch>
+    `,
+  )
+  .add(
+    'All text slots',
+    () => html`
+      <lion-input-switch label="Label" help-text="Help text">
+        <div slot="after">After</div>
+      </lion-input-switch>
+    `,
+  )
+  .add('Validation', () => {
+    const isTrue = value => value && value.checked && value.checked === true;
+    const isTrueValidator = (...factoryParams) => [
+      (...params) => ({
+        isTrue: isTrue(...params),
+      }),
+      ...factoryParams,
+    ];
+    const tagName = 'lion-switch-validation-demo';
+    if (!customElements.get(tagName)) {
+      customElements.define(
+        tagName,
+        class extends LocalizeMixin(LitElement) {
+          static get localizeNamespaces() {
+            const result = [
+              {
+                'lion-validate+isTrue': () =>
+                  Promise.resolve({
+                    info: {
+                      isTrue: 'You will not get the latest news!',
+                    },
+                  }),
+              },
+              ...super.localizeNamespaces,
+            ];
+            return result;
+          }
+
+          render() {
+            return html`
+              <lion-form id="postsForm" @submit="${this.submit}">
+                <form>
+                  <lion-input-switch name="emailAddress" label="Share email address">
+                  </lion-input-switch>
+                  <lion-input-switch name="subjectField" label="Show subject field" checked>
+                  </lion-input-switch>
+                  <lion-input-switch name="characterCount" label="Character count">
+                  </lion-input-switch>
+                  <lion-input-switch
+                    name="newsletterCheck"
+                    label="Subscribe to newsletter"
+                    .infoValidators="${[isTrueValidator()]}"
+                  >
+                  </lion-input-switch>
+                  <button type="submit">
+                    Submit
+                  </button>
+                </form>
+              </lion-form>
+            `;
+          }
+
+          submit() {
+            const form = this.shadowRoot.querySelector('#postsForm');
+            const validateElement = this.shadowRoot.querySelector('[name=emailAddress]');
+            validateElement.validate();
+            if (form.errorState === false) {
+              console.log(form.serializeGroup());
+            }
+          }
+        },
+      );
+    }
```

Comment:
An added demo that shows a disabled switch would be nice

Reply:
added, please confirm and "Resolve conversation"